### PR TITLE
All the neccessary stuff for deployment

### PR DESCRIPTION
**Pull request by zaqbez39me**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,21 +48,19 @@
 4. `python3 -m venv venv`
 5. `source venv/bin/activate`
 6. `pip install -r requirements.txt`
-7. `python3 manage.py migrate`
-8. `python3 manage.py makemigrations`
-9. `python3 manage.py makemigrations main`
-10. `python3 manage.py migrate`
+7. `python3 manage.py makemigrations`
+8. `python3 manage.py makemigrations main`
+9. `python3 manage.py migrate`
```
Comment (osx11) [inline README.md:54]: you should include 10. `python3 manage.py runserver`

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+case "$OSTYPE" in
+  linux*) sh linux-build.sh;;
+  msys*) sh windows-build.sh;;
+  cygwin*) sh windows-build.sh;;
+  *)      echo "unknown: $OSTYPE" ;;
+esac
```
Comment (osx11) [inline build.sh:7]: insert a new line

File: first-run.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    sh linux-runserver.sh;;
+  msys*)
+    source windows-build.sh
+    source windows-runserver.sh;;
+  cygwin*)
+    source windows-build.sh
+    source windows-runserver.sh;;
+  *)
+    echo "unknown: $OSTYPE" ;;
+esac
```
Comment (osx11) [inline first-run.sh:14]: insert a new line

File: linux-build.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+# shellcheck disable=SC1091
+
+sudo apt-get update
+sudo apt-get -y install libreoffice python3 python3-venv
+python3 -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+python3 manage.py makemigrations
+python3 manage.py makemigrations main
+python3 manage.py migrate
```
Comment (osx11) [inline linux-build.sh:11]: insert a new line

File: linux-runserver.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+source venv/bin/activate
+python manage.py runserver
```
Comment (osx11) [inline linux-runserver.sh:5]: insert a new line

File: main/windows.py [hunk 6]
```diff
@@ -56,9 +61,15 @@ def windows_convert_xlsx(excel, xlsx_filepath: str, pdf_filepath: str):
...
+    try:
+        xl_sheets = excel.Workbooks.Open(xlsx_filepath)
+        sheets_opened = True
+        xl_sheets.Worksheets[0].ExportAsFixedFormat(0, pdf_filepath)
+    finally:
+        if sheets_opened and xl_sheets is not None:
+            xl_sheets.Close(True)
 
 
 def windows(paths, files, has_type_in_request: dict):
```
Comment (osx11) [inline main/windows.py:75]: types of `paths` and `files` are not specified

Comment (zaqbez39me) [reply main/windows.py:75]: bracnh a bit depricated

File: runserver.sh [hunk 7]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 case "$OSTYPE" in
-  linux*) sh ./linux.sh;;
-  msys*) sh ./windows.sh;;
-  cygwin*) sh ./windows.sh;;
+  linux*) sh linux-runserver.sh;;
+  msys*) sh windows-runserver.sh;;
+  cygwin*) sh windows-runserver.sh;;
   *)      echo "unknown: $OSTYPE" ;;
 esac
```
Comment (osx11) [inline runserver.sh:7]: insert a new line

File: windows-build.sh [hunk 8]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+python -m venv venv
+source venv/Scripts/activate
+pip install -r requirements.txt
+python manage.py makemigrations
+python manage.py makemigrations main
+python manage.py migrate
```
Comment (osx11) [inline windows-build.sh:9]: insert a new line

File: windows-runserver.sh [hunk 9]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+source venv/Scripts/activate
+python manage.py runserver
```
Comment (osx11) [inline windows-runserver.sh:5]: insert a new line

**Review by osx11 — APPROVED** [review]

**Pull request merged**